packages/apollo-voyager-offline/src/conflictHandlers.ts

```diff
@@ -0,0 +1,63 @@
+import { SyncServerError } from '@aerogear/apollo-voyager-tools'
+import debug from 'debug'
+import { CONFLICT_ERROR, CONFLICT_LOGGER } from './constants'
+import { ObjectState } from './ObjectState'
+
+export const logger = debug(CONFLICT_LOGGER)
+
+/**
+ *
+ * @param helper the responsable to check the conflict and move the object to the next stage
+ * @param currentRecord the object state that the server knows about
+ * @param client the object state that the client knows about
+ */
+export const returnToClient: ConflictResolutionHandler = (helper: ObjectState, currentRecord: ConflictResolutionData, client: ConflictResolutionData): SyncServerError => {
```

Comment:
We are not following interface. ConflictResolutionHandler may not be that useful anymore. 
Need to think how to do that properly.